### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by ZachBourqueBCGov**
# Summary

Added the landing page as per Mary Kim's mockups

[](https://hamzamohdzubair.github.io/redant/)

- None listed.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] Dependency updates (may include refactoring of a dependencies functionality).
- [x] New feature (non-breaking change which adds functionality).

## Checklist:

- [x] I have read the checklist.

### CODE REVIEW

**Comment by ZachBourqueBCGov** [comment]
Currently, the "Login with IDIR" button is a placeholder, until @Bottle7's button styles have been merged in

File: matchmaking/frontend/src/theme.js [hunk 1]
```diff
@@ -12,8 +12,13 @@ const themeConfig = {
         fontColor: {
             white: "#fff",
             black: "#000"
-        }
+        },
     },
+    typography: {
+        fontFamily: [
+            'BC Sans'
```
Comment (Bottle7) [inline matchmaking/frontend/src/theme.js]:
```suggestion
            'BCSans'
```

File: matchmaking/frontend/index.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React</title>
   </head>
-  <body>
+  <body style="margin: 0;">
```
Comment (Bottle7) [inline matchmaking/frontend/index.html]: This inline style can be removed and, if necessary, added to the new base.css file that was added in my PR from earlier today.

**Review by Bottle7 — APPROVED** [review]
Fantastic work, you wizard!

**Pull request merged**